lib/utils.js

```diff
@@ -777,7 +777,7 @@ export function formatTasksData(
     if (inv.noteInvitation) {
       if (
         inv.details.repliedNotes?.length ||
-        (inv.apiVersion === 2 && inv.details.repliedEdits?.length)
+        (inv.apiVersion === 2 && inv.details.repliedEdits?.length && !inv.details.repliedEdits.slice(-1)[0].note.ddate)
```

Comment:
What if there was an invitation with `minReplies` > 1? Should we be counting note replies here?

Reply:
minReplies is used for notes not for edits. 

I don't think we are checking that in the code....

Reply:
Couldn't we iterate over the `repliedEdits` and get a count of the notes?

Reply:
I'm not sure about this approach because there can be multiple edits for the same note and we would need to check if the latest edit doesn't delete the note. 

AFAIK we are not using minReplies > 1 for note invitations so I would create an issue and fix this in another PR.